cypress-smoketests/integration/filings-and-qc.spec.js

```diff
@@ -0,0 +1,103 @@
+const {
+  addStatistic,
+  confirmServePetition,
+  goToMyDocumentQC,
+  goToPetitionNeedingQC,
+  goToPetitionNeedingQCByCaseTitle,
+  goToReviewPetition,
+  goToSectionDocumentQC,
+  savePetitionForLater,
+  selectCaseType,
+  selectTab,
+  servePetition,
+} = require('../support/pages/document-qc');
+const {
+  goToCaseDetail,
+  goToFileADocument,
+  goToFileYourDocument,
+  goToReviewDocument,
+  goToSelectDocumentType,
+  selectDocumentType,
+  submitDocument,
+  uploadDocumentFile,
+} = require('../support/pages/petitioner-dashboard');
+const { getUserToken, login } = require('../support/pages/login');
+
+let token = null;
+
+describe.skip('Petitions clerk', () => {
```

Comment:
@ericsorenson Just wanted to check if we have something logged for updating these so they're functioning/not skipped?

Reply:
Yes, skipped tests are flagged by our ESLint config. Going forward, any time we skip a test we should have a comment describing why.

Reply:
Added a comment!